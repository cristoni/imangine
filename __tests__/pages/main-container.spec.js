import React from 'react';
import MainContainer from '../../components/main-container';
import renderer from 'react-test-renderer';

test('MainContainer renders the main page container', () => {
  const component = renderer.create(
    <MainContainer />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});