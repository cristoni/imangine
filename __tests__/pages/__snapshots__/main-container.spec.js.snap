// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContainer renders the main page container 1`] = `
<div
  className="container"
/>
`;
